lib/datasource/local/models/substrate_extrinsic_model.dart

```diff
@@ -0,0 +1,83 @@
+/// From SDK TxInfoData - it's a model class that encodes a substrate extrinsic call
```

Comment:
just renamed from SubstrateTransactionModel - we now use transaction model name for this + parameters.